crypto/s2n_stream_cipher_rc4.c

```diff
@@ -21,20 +21,30 @@
 #include "utils/s2n_blob.h"
 #include "utils/s2n_safety.h"
 
-static uint8_t s2n_stream_cipher_rc4_available()
+static const EVP_CIPHER *s2n_evp_rc4()
```

Comment:
It's hard to tell from the docs but does this `EVP_CIPHER` need to be free'd with `EVP_CIPHER_free`? Or is it a global?